frontend/src/app/components/home/product/post/post-product.component.html

```diff
@@ -25,22 +28,32 @@ <h3 class="title">Create</h3>
       <div class="input-grid temp-spaceing">
         <label class="picture">
           Picture
+          <div *ngIf="isUpdate">
+            <img class="image" src={{picture}}>
+            <div class="imageText">Do you want to Update the Image:</div>
+          </div>
           <input (change)="selectFile($event)" name="picture" type="file" size="50" accept="png, jpg">
         </label>
         <input placeholder="Expiration Date" name="expirationDate" [(ngModel)]="values.expirationDate" type="datetime-local" appDateValidator>
       </div>
-      <div class="action-wrapper">
+      <div class="action-wrapper" *ngIf="!isUpdate, else updateButton">
         <button [ngClass]="'button-empty-' + theme" (click)="showPreview(postProductForm, preview)" type="button">Preview</button>
         <button [ngClass]="'button-' + theme" type="submit"  [disabled]="!postProductForm.valid">Create</button>
       </div>
+      <ng-template #updateButton> 
```

Comment:
I would suggest putting the ngIf inside the action wrapper, because you have the same preview button but the create button changes to a update button

Reply:
tried this also but it broke the layout somehow